functions/fn_initServer.sqf

```diff
@@ -34,5 +34,11 @@ params ["_logicModule"];
 // init finished, broadcast
 _logicModule setVariable ["SCAR_UCM_initialized", true, true];
 
+//create worker groups for every sife
```

Comment:
I have no idea if this is a legitimate way of creating sides. It would likely be better to add the workers to a group based on the UCM Logic module